03-client-api/01-java.md

```diff
@@ -10,6 +10,7 @@ templatePath: 03-client-api/references/
 
 | Client Java | Grakn Core     | Grakn KGMS     |
 | :---------: | :-------------:| :------------: |
+| 1.7.2       | 1.7.1          | N/A            |
```

Comment:
I don't think we need to list it as supporting 1.7.0, minor versions should nearly always be latest (as they should contain bug fixes).

Reply:
our past style indicates all the compatible versions, including minor I believe. This might be up for change at some point when we're more stable though!